sharktuner/sharktuner/common.py

```diff
@@ -214,6 +214,11 @@ class AttentionOpInfo:
 
 @dataclass
 class LLVMGPUVectorDistributeContractionKnobs(KnobAssignment):
+    # Problem Size
+    m: int
+    n: int
+    k: int
```

Comment:
We use uppercase MNK for problem size in the compiler
```suggestion
    M: int
    N: int
    K: int
```

Reply:
[c964648](https://github.com/nod-ai/shark-ai/pull/2555/commits/c964648f7d907d3a87a371a8250082189c394931)